IdentityCore/src/configuration/MSIDNetworkConfiguration.m

```diff
@@ -23,41 +23,28 @@
 
 #import "MSIDNetworkConfiguration.h"
 
-static NSTimeInterval const s_defaultTimeout = 30;
-static int const s_defaultRetryCount = 2;
+static NSTimeInterval s_timeout = 30;
+static NSInteger s_retryCount = 2;
 
 @implementation MSIDNetworkConfiguration
 
-- (instancetype)init
++ (void)setTimeout:(NSTimeInterval)timeout
 {
-    self = [super init];
-    if (self)
-    {
-        _timeout = s_defaultTimeout;
-        _retryCount = s_defaultRetryCount;
-    }
-    return self;
+    s_timeout = timeout;
```

Comment:
Maybe it is better to have default _timeout_ & _retryCount_ setter & getters and assign default values to them in initializer? In such case you will not need to override setters & getters.

Reply:
these are class properties, which means they don't have default synthesizer.
thus, we'd need to write a setter/getter anyways. so, with that, I don't see a need for an initializer.